packages/bpk-component-select/README.md

```diff
@@ -34,8 +34,8 @@ export default () => (
 | ------------              | --------   | -------- | ------------- |
 | id                        | string     | true     | -             |
 | name                      | string     | true     | -             |
-| value                     | string     | true     | -             |
-| onChange                  | func       | true     | -             |
+| value                     | string     | false    | use [defaultValue](https://reactjs.org/docs/dom-elements.html#value)|
+| onChange                  | func       | false    | undefined     |
```

Comment:
If these are no longer required, we will probably need to modify the examples above to not include these as they are optional.